smart_campus/smart_campus/settings/base.py

```diff
@@ -30,9 +30,6 @@ def get_env_variable(var_name):
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
 
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = get_env_variable('SECRET_KEY')
```

Comment:
@jeffwang0516 
Do not remove `SECRET_KEY` so that other config can fall back to `base.py` and load `SECERT_KEY` from env variable.

Reply:
Got a question here.
Is it correct that I still have to set environment variables even if I runserver with local_settings ?
local_settings.py do not use any get_env_variable().

Reply:
You should set all the variables that are needed across different setting files in `base.py`.
However, you can overwrite these variables in other setting files (e.g. `local_setting.py`).
If you forget or don't want to hardcode some in these setting files, they could still be loaded from environment variables.

Reply:
But in the setting file(local), I did overwrite those variables.
It seems that it executes  "SECRET_KEY = get_env_variable()" in base.py and then "SECRET_KEY=" hardcode value "  in local_settings.py
Raising exception ImproperlyConfigured from the former.

Reply:
All, my mistake.
It works on "flask."
I've tried and found it didn't work.
I find a way to tackle this problem.
Thanks for this!

Reply:
@jeffwang0516 
After doing some survey on this issue, I find that you're right about it.
In "flask," the configuration is loaded as a class object instead of a statement.
Thus, they are overwritten before execution.

Feel free to remove those variables in `base.py`.
Thanks!

Reply:
Okay, got it.